care/emr/management/commands/load_fixtures.py

```diff
@@ -0,0 +1,433 @@
+from care.emr.resources.user.spec import UserCreateSpec
+from care.security.models import RoleModel
+from care.users.models import User
+from care.utils.tests.base import CareAPITestBase
+
+
+def generate_unique_indian_phone_number():
+    max_attempts = 1000
+    attempts = 0
+    while True:
+        attempts += 1
+        if attempts > max_attempts:
+            raise RuntimeError(
+                "Failed to generate unique phone number after maximum attempts"
+            )
+        number = 6000000000 + secrets.randbelow(9999999999 - 6000000000 + 1)
+        phone_str = f"+91{number}"
+        try:
+            parsed_number = phonenumbers.parse(phone_str, "IN")
+            if phonenumbers.is_valid_number_for_region(parsed_number, "IN"):
+                result = phonenumbers.format_number(
+                    parsed_number, PhoneNumberFormat.E164
+                )
+                if User.objects.filter(phone_number=result).exists():
+                    continue
+                return result
+        except NumberParseException:
+            continue
+
+
+class Command(BaseCommand):
+    help = "Generate test fixtures for the backend"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--users", type=int, default=1, help="Number of each type of users"
+        )
+        parser.add_argument(
+            "--patients", type=int, default=10, help="Number of patients"
+        )
+        parser.add_argument(
+            "--encounter", type=int, default=1, help="Number of encounters per patient"
+        )
+        parser.add_argument(
+            "--default-password",
+            type=str,
+            default="Coronasafe@123",
+            help="Set a default password for all users (easier for testing)",
+        )
+
+    def handle(self, *args, **options):
+        if not settings.DEBUG:
+            self.stdout.write(
+                self.style.ERROR(
+                    "This command should not be run in production. Exiting..."
+                )
+            )
+            return
+
+        self.stdout.write("Starting fixtures generation...")
+
+        self.stdout.write("Syncing permissions and valuesets...")
+        call_command("sync_permissions_roles")
+        call_command("sync_valueset")
+
+        try:
+            with transaction.atomic():
+                self._generate_fixtures(options)
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        "Successfully generated all fixtures in transaction!"
+                    )
+                )
+        except Exception as e:
+            self.stdout.write(
+                self.style.ERROR(f"Transaction rolled back due to error: {e}")
+            )
+            raise
+
+    def _generate_fixtures(self, options):
+        """Generate all the fixture data within a transaction context."""
+        base = CareAPITestBase()
+        fake = Faker()
```

Comment:
```suggestion
        fake = Faker("en_IN")
```
lets use the indian dataset